src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -704,6 +711,8 @@ public void PlaceOrganelle()
             // Only trigger tutorial if something was really placed
             TutorialState.SendEvent(TutorialEventType.MicrobeEditorOrganellePlaced, EventArgs.Empty, this);
         }
+
+        HandleIslands();
```

Comment:
This needs to also trigger when undoing / redoing. This should be moved to where the other actions happen as well after action undo / redo.